modules/openCypher-in-gsql/pages/openCypher-in-gsql.adoc

```diff
@@ -324,9 +324,9 @@ WHERE x.friendCount = y.friendCount
 
 * Pattern fragments (x:user) and (y:user) *are not* connected by edge traversal or by sharing vertex variables.
 
-== Key Differences: GSQL vs. OpenCypher
+== Key Differences: GSQL OpenCypher mode vs. OpenCypher
 
-GSQL is strongly typed and **schema-first**, while OpenCypher is **schema-optional**. This fundamental difference impacts how OpenCypher queries behave in each system.
+GSQL OpenCypher is strongly typed and **schema-first**, while OpenCypher is **schema-optional**. This fundamental difference impacts how OpenCypher queries behave in each system.
```

Comment:
"GSQL OpenCypher" sounds strange to me because GSQL is a graph query language (and more) , and OpenCypher is another graph query language.  The direct joining of the two words doesn't make semantic sense. It's like saying "Toyota Honda car". Previously on this page, we used the more explicit phrases "OpenCypher in GSQL" and "GSQL OpenCypher mode." For the later, it would be even more appropriate to say "GSQL's OpenCypher mode" (use the possession form of GSQL.)

When you want to define a new, briefer terminology, the standard method is to give the full name, then the abbreviate term in parentheses and quotation marks, such as

GSQL's OpenCypher mode ("GSQL OpenCypher") is strongly type.